### PR TITLE
Add outreachy'22 wrap-up blog

### PR DESCRIPTION
**Pull request by ssurbhi560**
Adding the outreachy wrap-up blog with a brief description of all the tickets that were solved during the internship.

### CODE REVIEW

**Comment by ssurbhi560** [comment]
Hi @jaimergp, the blog is done! :)

File: posts/2022-08-19-outreachy-wrap-up-blog-2022.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.. post:: 2022-08-19
+  :tags: Outreachy
+  :author: Surbhi Sharma
+  :redirect: 2022/08/19/outreachy-wrap-up-blog-2022
+
+Outreachy 2022 Wrap-up Blog
+============================
+
+This blog is about my work during my Outreachy internship with `conda-forge <https://conda-forge.github.io>`__. Before that a little about me - 
+I am `Surbhi <https://github.com/ssurbhi560>`__, an outreachy intern with conda-forge for the May-August 2022 cohort and I worked on documenting the conda-forge ecosystem.
```
Comment (jaimergp) [inline posts/2022-08-19-outreachy-wrap-up-blog-2022.rst]:
```suggestion
I am `Surbhi <https://github.com/ssurbhi560>`__, an Outreachy intern with conda-forge for the May-August 2022 cohort and I worked on documenting the conda-forge ecosystem.
```

File: posts/2022-08-19-outreachy-wrap-up-blog-2022.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+        With these, we added more documentation on migrations and migrators, which would help maintainers find answers to questions like - What is a migrator/migration, and what does it do? When can (and why would) they should reject a migration PR? And so on.
+
+10. `Add a section in docs on security aspects of conda-forge (#1808) <https://github.com/conda-forge/conda-forge.github.io/issues/1808>`__. Working on this `#1812 <https://github.com/conda-forge/conda-forge.github.io/pull/1812>`__.
+
+        Currently, the documentation regarding the security considerations of conda-forge builds is scattered throughout the documentation, and therefore it is hard to find and read it. With this issue, we will put all the information together in one place, which will help maintainers and users to know more about how conda-forge secures its packages.
```
Comment (jaimergp) [inline posts/2022-08-19-outreachy-wrap-up-blog-2022.rst]:
```suggestion
        Currently, information regarding the security considerations of conda-forge builds is scattered throughout the documentation, and therefore it is hard to find and read. With this issue, we will put all the information together in one place, which will help maintainers and users to know more about how conda-forge secures its packages and infrastructure.
```

**Review by jaimergp — APPROVED** [review]
I fixed a little issue with the CI, and added a couple of (optional) suggestions for the text. LGTM!

Pinging @conda-forge/core and @kathatherine in case anybody else wants to take a look.

File: posts/2022-08-19-outreachy-wrap-up-blog-2022.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
+.. post:: 2022-08-19
+  :tags: Outreachy
+  :author: Surbhi Sharma
+  :redirect: 2022/08/19/outreachy-wrap-up-blog-2022
```
Comment (jaimergp) [inline posts/2022-08-19-outreachy-wrap-up-blog-2022.rst]:
```suggestion
.. post:: 2022-08-26
  :tags: Outreachy
  :author: Surbhi Sharma
  :redirect: 2022/08/26/outreachy-wrap-up-blog-2022
```

Let's fix the date (assuming we merge tomorrow) on this metadata block _and_ the filename.

**Comment by ssurbhi560** [comment]
> I fixed a little issue with the CI, and added a couple of (optional) suggestions for the text. LGTM!

The commits Jaime made to fix the CI got deleted because of some mess I did with git. Sorry for that; I have pushed those changes again now.

File: posts/2022-08-19-outreachy-wrap-up-blog-2022.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+5.  `DOC: New maintainer (#1117) <https://github.com/conda-forge/conda-forge.github.io/issues/1117>`__. Closed with `Add “How regro-cf-autotick-bot create version update PR?” section. (#1788) <https://github.com/conda-forge/conda-forge.github.io/pull/1788>`__.
+
+        With this issue we improved docs for the new maintainers and the working of the bot. A “How does ``regro-cf-autotick-bot`` create automatic version updates?” section was added to the docs, which explains the whole process of creating an automated version update PRs by bot.
+
+6.  `Add Perl package hints to documentation (#1536) <https://github.com/conda-forge/conda-forge.github.io/issues/1536>`__. Working on this `#1790 <https://github.com/conda-forge/conda-forge.github.io/pull/1790>`__.
+
+        With this issue we added ​​packaging instructions for Perl packages with different build systems in the documentation.
+
+7.  `DOC: Update documentation about tokens (#1532) <https://github.com/conda-forge/conda-forge.github.io/issues/1532>`__. Working on this `#1793 <https://github.com/conda-forge/conda-forge.github.io/pull/1793>`__.
```
Comment (kathatherine) [inline posts/2022-08-19-outreachy-wrap-up-blog-2022.rst]:
```suggestion
7.  `DOC: Update documentation about tokens (#1532) <https://github.com/conda-forge/conda-forge.github.io/issues/1532>`__. Closed with `#1793 <https://github.com/conda-forge/conda-forge.github.io/pull/1793>`__.
```

File: posts/2022-08-19-outreachy-wrap-up-blog-2022.rst [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+        With this issue we improved the documentation on ``arch_rebuild.txt`` and how maintainers can add a feedstock to ``arch-rebuild.txt`` if it requires rebuilding with different architectures/platforms (such as ppc64le or aarch64).
+
+9.  `Document migrators (#1355) <https://github.com/conda-forge/conda-forge.github.io/issues/1355>`__, `Update migration docs (#862) <https://github.com/conda-forge/conda-forge.github.io/issues/862>`__, and `document migrators (#737) <https://github.com/conda-forge/conda-forge.github.io/issues/737>`__ . Closed with `Documenting Migrators and Migrations. (#1801) <https://github.com/conda-forge/conda-forge.github.io/pull/1801>`__.
+
+        With these, we added more documentation on migrations and migrators, which would help maintainers find answers to questions like - What is a migrator/migration, and what does it do? When can (and why would) they should reject a migration PR? And so on.
+
+10. `Add a section in docs on security aspects of conda-forge (#1808) <https://github.com/conda-forge/conda-forge.github.io/issues/1808>`__. Working on this `#1812 <https://github.com/conda-forge/conda-forge.github.io/pull/1812>`__.
```
Comment (kathatherine) [inline posts/2022-08-19-outreachy-wrap-up-blog-2022.rst]:
```suggestion
10. `Add a section in docs on security aspects of conda-forge (#1808) <https://github.com/conda-forge/conda-forge.github.io/issues/1808>`__. Closed with `#1812 <https://github.com/conda-forge/conda-forge.github.io/pull/1812>`__.
```

**Review by kathatherine — CHANGES_REQUESTED** [review]
Looks great! Just make sure those two issues that we closed in the last week are marked as such in the blog.

**Review by kathatherine — APPROVED** [review]
LGTM!

**Comment by kathatherine** [comment]
@beckermr @jaimergp Shall we go ahead and merge this?

**Comment by jakirkham** [comment]
Thanks Surbhi! Also thanks all for reviewing 🙏

**Comment by jakirkham** [comment]
cc @tnabtaf (for awareness)

**Comment by jakirkham** [comment]
Noticed a small formatting thing. Submitted PR ( https://github.com/conda-forge/blog/pull/58 ) to adjust that

**Pull request merged**